deps/fs.lua

```diff
@@ -180,9 +180,9 @@ local function readdir(path, callback)
     local files = {}
     local i = 1
     while true do
-      local ent = uv.fs_scandir_next(req)
-      if not ent then break end
-      files[i] = ent.name
+      local name = uv.fs_scandir_next(req)
+      if not name then break end
+      files[i] = name
```

Comment:
The only problem here is lit packages have no way to signify what versions of luv they depend on.  If we publish a new fs that uses the new style from luvi 2.6.0, then anyone using an older version will get broken.